lib/datasets.js

```diff
@@ -17,6 +17,27 @@ function getData(datasetId) {
   return datasetsDatabase.get(`${datasetId}-data`)
 }
 
+ function getStats(datasets) {
+  const stats = {
+    count: datasets.length,
+    license: {
+      lov2: 0,
+      'odc-odbl': 0
+    },
+    model: {
+      'bal-aitf': 0,
+      custom: 0
+    }
+  }
+
+  datasets.forEach(dataset => {
+    dataset.license === 'lov2' ? stats.license.lov2 += 1 : stats.license['odc-odbl'] += 1
```

Comment:
La ternaire nécessitera inévitablement une réécriture en cas de nouvelle valeur.
Pourquoi ne pas simplement utiliser incrémenter ou mettre à 1 si `undefined` ?